src/App.js

```diff
@@ -1,20 +1,20 @@
-import React from "react"
-import { BrowserRouter as Router } from "react-router-dom"
-
-import BaseRouter from "./routes"
-import { ThemeProvider } from "styled-components"
-import { globalStyles } from "./globalStyles"
+import React from 'react';
+import { BrowserRouter, Router } from 'react-router-dom';
```

Comment:
Codacy found an issue: ['BrowserRouter' is defined but never used.](https://app.codacy.com/gh/codeclannigeria/codeclannigeria-frontend/pullRequest?prid=5730659)